Controller/Console.java

```diff
@@ -0,0 +1,7 @@
+package Controller;
+
+public class Console {
```

Comment:
I like something like Parser for the name unless we are doing more with this class that relates to console as a name later on

Reply:
Fixed in https://github.com/BrosinTimeCode/RTS/pull/17/commits/375d6c49ca50715182ac54321a514c6e0c8d9773